src/hct_mis_api/apps/program/fixtures.py

```diff
@@ -140,3 +140,74 @@ def get_program_with_dct_type_and_name(
         **kwargs,
     )
     return program
+
+
+def generate_beneficiary_groups() -> None:
+    BeneficiaryGroupFactory(
```

Comment:
@pavlo-mk same as above

Reply:
this one as well used only in` initdemo.py`